pages/static/data-updates.html

```diff
@@ -4,6 +4,20 @@
 ---
 
 <h1>Data Updates</h1>
+<h2 id="2023-03-10">March 10th, 2023</h2>
+<p>New data is available from WUSTL and MSK atlases as part of Release 3.1:</p>
+<ul>
+    <li>Level 1 & 2 10X Visium data from WUSTL PDAC and CCRCC studies</li>
+    <li>
+        Level 2 & 3 single cell and single nucleus RNA-sequencing data from
+        WUSTL PDAC study
+    </li>
+    <li>
+        Additional Level 1 single cell RNA-sequencing files from WUSTL PDAC
```

Comment:
Change to: 
"Additional Level 1-3 single cell RNA-sequencing files from WUSTL PDAC study"

WUSTL added a few higher level ScRNA-seq files for the `HTA12_6_13` sample on Wednesday, so updating to reflect that

Reply:
In that case, maybe I combine the two lines then?

> Level 1- 3 single cell and single nucleus RNA-sequencing data from WUSTL PDAC study

Reply:
Thanks, lemme double check with Vesteinn really quickly. 
The intention was to distinguish between their 2 PDAC submissions, may re-work it a little bit

Reply:
Ok, let's leave it as is :) Thank you!